openwisp_controller/subnet_division/admin.py

```diff
@@ -21,10 +21,24 @@
 
 
 class SubnetDivisionRuleInlineAdmin(
-    MultitenantAdminMixin, TimeReadonlyAdminMixin, admin.StackedInline
+    MultitenantAdminMixin, TimeReadonlyAdminMixin, HelpTextStackedInline
 ):
     model = SubnetDivisionRule
     extra = 0
+    help_text = {
+        'text': _(
+            'Please keep in mind that once the subnet division rule is created '
+            'changing changing "Size", "Number of Subnets" or decreasing '
+            '"Number of IPs" will not be possible.'
+        ),
+        'documentation_url': (
+            'https://github.com/openwisp/openwisp-controller/tree/master'
```

Comment:
please omit `tree/master`, we generally don't use it in these kind of links and we should be consistent.